### DISCUSSION TITLE
How do you check if the video id is currently playing a live stream or a VOD?

### DISCUSSION THREAD

**Discussion by nick-tw** [vimeo/player.js#666] [General]
I'm building a frontend to embed a Vimeo player for a live conference. The frontend will render the UI different if the live stream video has ended and became a VOD video.

I've looked through the API but there doesn't seem to be a method to check whether a video ID is currently a live stream or if it's a replay of the event.

**Comment by fisherinnovation** [comment]
Thanks for the feedback. I do see the value of having some level of indication of the stream type. Marking as a feature request for now.

**Comment by rondog** [comment]
I just made a fiddle and tested the getDuration method on a live video and it returned 0 while a VOD video returned an actual number. You may be able to get something working using that.

**Comment by ghost** [comment]
It would be really nice if this feature will be available for us. Is there any progress on this?
